### DISCUSSION TITLE
auto import for useShallow is not working.

### DISCUSSION THREAD

**Discussion by YJ-AnthonyJo** [pmndrs/zustand#2204] [Q&A]
When I try to use useShallow, intellisense not auto import it.

### My expectation:
like useState or other functions from libraries, I wanted to import functions with auto import
<img width="939" alt="image" src="https://github.com/pmndrs/zustand/assets/41505000/88a722a4-6264-4514-97f8-8f68d10a4ff4">

### Actual behavior
<img width="339" alt="image" src="https://github.com/pmndrs/zustand/assets/41505000/2db65ff9-299c-4626-a8ae-e0f367d89200"> 

nothing shows..

**Comment by dai-shi** [comment]
Maybe it doesn't work for non-default exports.
For example, does it work for `shallow` or `persist`?

**Reply by YJ-AnthonyJo** [reply]
No `shallow` and `persist` are not auto imported or suggested.

But, auto import of `useShallow` is now working..
I didn't change anything..
(but still `shallow`, `persist` don't work)

Do you have any idea of these behavior? just curiosity.
Thank you for reply.🙏

**Reply by dbritto-dev** [reply]
@YJ-AnthonyJo  let me look into that

**Reply by YJ-AnthonyJo** [reply]
@dbritto-dev Thanks!

**Reply by YJ-AnthonyJo** [reply]
For more info, When I tried to `import {useShallow} from 'zustand/react/shallow'`,
There were no suggestion of `zustand/react` when I just typed `zustand/`.
And it is still happening after when i'm [able to auto import `useShallow`. ](https://github.com/pmndrs/zustand/discussions/2204#discussioncomment-7632762)
I'm using v4.4.6.

<img width="500" alt="image" src="https://github.com/pmndrs/zustand/assets/41505000/2c35228b-7684-45b9-89e0-c22362ba174a">

**Reply by dbritto-dev** [reply]
@YJ-AnthonyJo seems related to exports, we're planning to fix that on v5

**Reply by YJ-AnthonyJo** [reply]
@dbritto-dev Well I found the reason.. And I want to contribute to this project but I'm not sure is it okay to contribute as a junior.. May I request PR to zustand project?

**Reply by dbritto-dev** [reply]
@YJ-AnthonyJo don't worry about that let's open the PR. I would be happy to review it and help you with that. I had a fix too but that requires to update some stuff  that we are planning to update on v5.

**Reply by YJ-AnthonyJo** [reply]
Thank you for your kindness! I'll open the PR as soon as possible.
